src/process.h

```diff
@@ -39,6 +39,8 @@ struct cmd_t {
  *    filename -- The output of the commands are appended to filename
  */
 struct ctx_t {
+  ctx_t() : include_global_prep(true) {}
```

Comment:
Preferably booleans should initialize at false, it's not a hard requirement but that is general convention. I would invert the boolean here to state disable_global_prep instead. That way you don't have to add an initializer and it matches with most of the booleans in this code base.

Reply:
I think they just didn't get around to updating it yet... We had the same discussion here: https://discord.com/channels/804382334370578482/1074377411069747250/1077688007659896832

Reply:
Fixed in latest commit.